lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,167 @@
-                display: none;
-              }
-
-              // definitions for proctored exam attempt status indicators
-              i.verified {
-                color: $success-color;
-              }
-
-              i.rejected {
-                color: $alert-color;
-              }
-
-              i.error {
-                color: $alert-color;
-              }
-            }
-          }
-
-          &:hover, &:focus {
-            background: $shadow-l1;
-
-            > a p {
-              color: $gray-d3;
-            }
-          }
-
-          &:active {
-            box-shadow: inset 0 1px 14px 0 $shadow-l1;
-
-            &:after {
-              opacity: 1.0;
-              right: 15px;
-            }
-          }
-        }
-
-        &.active {
-          @extend %t-weight5;
-
-          &:after {
-            content: '›';
-            position: absolute;
-            top: 50%;
-            right: 20px;
-            margin-top: -13px;
-            font-size: 30px;
-            font-weight: normal;
-            color: #333;
-            opacity: 0;
-            @include transition(none);
-          }
-
-          > a {
-            border: 1px solid $border-color-1;
-            box-shadow: 0 1px 0 rgba(255, 255, 255, .35) inset;
-            background: $sidebar-active-image;
-
-            &:after {
-              opacity: 1.0;
-              right: 15px;
-            }
-
-            p {
-              color: #333;
+    .accordion {
+        @extend %t-copy-sub1;
+
+        .course-navigation {
+
+            .button-chapter {
+                @include box-sizing(border-box);
+                @include linear-gradient(top, $sidebar-chapter-bg-top, $sidebar-chapter-bg-bottom);
+                @include transition(background-color .1s linear 0s);
+                width: 100% !important;
+                padding: 0;
+                border: 0;
+                border-radius: 0;
+                box-shadow: 0 1px 0 $white inset, 0 -1px 0 $shadow-l1 inset;
+                background-color: $sidebar-chapter-bg;
+                color: $link-color;
+
+                &.is-open {
+                    background: $white;
+                    box-shadow: none;
+                }
+
+                &.active {
+
+                    .group-heading {
+                        @extend %t-ultrastrong;
+                        color: $base-font-color;
+                    }
+                }
+
+                .group-heading {
+                    @extend %t-title6;
+                    position: relative;
+                    padding: ($baseline*.75) $baseline ($baseline*.75) ($baseline*2);
+                    margin: 0;
+                    border-radius: 0;
+                    box-shadow: none;
+                    color: $link-color;
+
+                    &:first-child {
+                        border: none;
+                    }
+
+                    .icon {
+                        position: absolute;
```

Comment:
* the absolute positioning is to offset it to the left of the text
* the font-size isn't necessary as it's handled in the `.fa` class for the icon font; I've removed it